wctl/tx.go

```diff
@@ -83,7 +83,7 @@ func (c *Client) SendTransaction(tag byte, payload []byte) (*TxResponse, error)
 	var res TxResponse
 
 	nonce := atomic.AddUint64(&c.Nonce, 1)
-	block := atomic.LoadUint64(&c.Block) + 1
+	block := atomic.LoadUint64(&c.Block)
```

Comment:
I thought when we add a transaction, block height is set to the next block that we are finalizing on, which is latest known block height + 1?

Reply:
The block height isn't strict - its just to tell other nodes the latest block height they are on when they proposed the tx.